package.json

```diff
@@ -20,7 +20,8 @@
   },
   "homepage": "https://github.com/lhvk/lars-halvor-kallak-css-frameworks-ca#readme",
   "dependencies": {
-    "bootstrap": "^5.2.2"
+    "bootstrap": "^5.2.2",
+    "bootstrap-icons": "^1.9.1"
```

Comment:
Load these from node_modules instead of CDN

Reply:
How do I do that?

Reply:
You need to import `bootstrap.scss` from node_modules into your main SCSS file.

In your HTML you will link to `/dist/css/styles.css` which is the output from SASS.